cmd/snap-seccomp/main_test.go

```diff
@@ -847,6 +847,42 @@ func (s *snapSeccompSuite) TestRestrictionsWorkingArgsTermios(c *C) {
 	}
 }
 
+func (s *snapSeccompSuite) TestRestrictionsWorkingArgsSyslog(c *C) {
+	for _, t := range []struct {
+		seccompAllowlist string
+		bpfInput         string
+		expected         int
+	}{
+		// good input
+		{"syslog SYSLOG_ACTION_SIZE_UNREAD", "syslog;native;SYSLOG_ACTION_SIZE_UNREAD", Allow},
+		{"syslog SYSLOG_ACTION_CLOSE", "syslog;native;0", Allow},
```

Comment:
Should this and the following `Allow` test cases use the named identifier rather than the 0,1,2 .. etc magic numbers?

Reply:
Ah yes this is left over from one of my previous iterations, will tweak